meta-lmp-base/recipes-bsp/u-boot/u-boot-ostree-scr-fit/boot-common.cmd.in

```diff
@@ -97,6 +98,18 @@ if test "${fiovb.debug}" = "1"; then
 	echo "${fio_msg} ###########################################"
 fi
 
+if test -n "${fiovb_rpmb}"; then
+	if test "${fiovb.rollback_protection}" = "1"; then
+		echo "${fio_msg} Anti-rollback protection for boot firmware is enabled"
+		if test ${fiovb.bootfirmware_version} -gt ${dt_bootfirmware_version}; then
```

Comment:
While not really safe when stored outside rpmb, should we also add rollback protection via u-boot env? (the user would be able to replace the content, but we could then at least validate the same ak-lite logic later).